### DISCUSSION TITLE
不能设置一些快捷键？比如那个。

### DISCUSSION THREAD

**Discussion by linxu2015** [hiroi-sora/Umi-OCR#639] [Ideas]
比如那个截图提取文字，然后复制好文字它就可以自动关闭。多一些功能键，还有设置优化。

**Comment by linxu2015** [comment]
截图识别    AIT+S

**Comment by hiroi-sora** [comment]
> 复制好文字它就可以自动关闭

截图OCR页 → 设置 → 识图后的操作 → 勾选 `复制结果` ，取消勾选 `弹出主窗口` 即可。
